api/v1/openapi.yaml

```diff
@@ -3,7 +3,7 @@ swagger: '2.0'
 info:
   title: Cilium API
   description: Cilium
-  version: v1beta
+  version: v1beta1
```

Comment:
Why is it necessary to change the version number? I think this is probably OK, but it could be helpful to have this in a separate dedicated commit with the rationale, that way we can try to ensure that any future changes to the version string can match whatever thing you had trouble with here.

Reply:
Since there was a change to the API (the addition of the SCTP stuff), I thought it would make sense to bump it. If that isn't the protocol, we can revert this change.

Reply:
Overall on this unix-socket API, there is no intention to make it a public, stable API. The stable APIs are typically the ones that interface with Kubernetes, structures we store in key-value stores, or the Hubble flow APIs. Generally we won't break things on this API if there's a reasonable solution to change something without breaking it, but for instance in this case it seems like there's just new numbers & fields that should be backwards-compatible anyway.

If CI's happy with this then I'm OK with it. I see it's in the same commit as the other API changes anyway so that'll do for now.